libminifi/include/utils/ThreadPool.h

```diff
@@ -170,7 +142,7 @@ template<typename T>
 class WorkerComparator {
  public:
   bool operator()(Worker<T> &a, Worker<T> &b) {
-    return a.getTimeSlice() < b.getTimeSlice();
+    return a.getTimeSlice() > b.getTimeSlice();
```

Comment:
I don't mind breaking API to a certain extent to fix a critical bug as this is, but this breaks API in a dangerous way: it will compile, but do the opposite of what it did so far.
So far this has been the comparator for the `worker_priority_queue_`, now it is used for the `delayed_worker_queue_`.
I would prefer if this functor was renamed `DelayedWorkerComparator`, so that it would break API in a safe way and would be more straightforward.

Reply:
Done